open_dread_rando/files/lua_libraries/room_names.lua

```diff
@@ -0,0 +1,527 @@
+RoomNameGui = RoomNameGui or {
+    cameraDict = nil
+}
+
+function RoomNameGui.BuildCameraDict()
+	if RoomNameGui.cameraDict ~= nil then
```

Comment:
Let's be consistent and not use tabs for this file :)

(I'll admit the `death_counter.lua` and `guilib.lua` uses tabs, so we're already a bit inconsistent oops)

Reply:
that's my fault sorry I'm a tab lover